[DM-710] updated user documentation of Actility and Sigfox device registration (#533)
SoftwareAG/c8y-docs: content/protocol-integration/sigfox-bundle/register-device-sigfox.md

---

> To register a Sigfox device, navigate to the **Registration** page in the **Devices** menu in the Device Management application and click **Register devices**. In the dropdown, select **Sigfox**.

Reviewer: ```suggestion
To register a Sigfox device, navigate to the **Registration** page in the **Devices** menu in the Device Management application and click **Register devices**. In the dropdown, select **Sigfox**.
To register a Sigfox device in {{< product-c8y-iot >}} navigate to **Devices** > **Registration** in the Device Management application, click **Register device** at the top right and select **Sigfox** from the dropdown.
```

Author: Corrected.

- **ID:** Unique device ID. The value must be a hexadecimal number.
- **PAC:** Porting authorization code for your device. The value must be a hexadecimal number.

> - **Connection**: It will list all the configured Sigfox connections in the tenant. The following contract option is populated based on selected Sigfox connection.

Reviewer: ```suggestion
- **Connection**: Lists all configured Sigfox connections in the tenant. The following contract option is populated based on the selected Sigfox connection.
```

Author: Corrected.

> - **Contract:** Select your desired contract (all the contracts are listed including active and expired).

Author: Corrected.

- **Device protocol:** Select your desired device protocol from the drop-down list.
- **Product certificate key:** This key can be located in *https://partners.sigfox.com/*. Navigate to your device and copy the certificate key. If the checkbox is not selected and no product certificate key is specified, the device will be considered a prototype.

{{< c8y-admon-info >}}
The term "Device type" is used both by Sigfox and {{< product-c8y-iot >}}, but with different meaning. In Sigfox, a device type specifies how to route data from devices. In {{< product-c8y-iot >}}, a device type describes the data that is sent by devices of a particular type.
{{< /c8y-admon-info >}}

![Register devices1](/images/device-protocols/sigfox/sigfox-registration1.png)

> After clicking **Register** the device registration request will be submitted and the device will be created.

Reviewer: ```suggestion
Click **Register** to submit the device registration request and create the device.
```

Author: Corrected.
